corehq/apps/linked_domain/dbaccessors.py

```diff
@@ -32,6 +32,17 @@ def is_active_downstream_domain(domain):
     return DomainLink.objects.filter(linked_domain=domain).exists()
 
 
+def is_active_link(upstream_domain, downstream_domain):
```

Comment:
I don't think this function is pulling its weight. It's only called once, and that call could be replaced with a call to `get_active_domain_link`. Plus you have to patch it multiple times.

Reply:
This is true. I added it specifically be able to patch it, so your observation checks out! I agree with you that I can just use `get_active_domain_link`.